whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
 
 Achieving the ultimate sensitivity of the LSST to weak lensing science places
-stringent requirements on our ability to accurately measure galaxy shapes and redshifts,
-which in turn demands precise and accurate knowledge of the point spread function,
-astrometry, and photometry. These measurements are influenced by the interaction of
-light with the Earth’s atmosphere, the telescope optics, and the CCD sensors. Sysematics
-in the shear are introduced in each case.   Methods have been developed for suppressing 
-these systematics in current lensing surveys. These and new methods will be applied to 
-the LSST survey.
-
-Over the sample of 3-4 billion galaxies, the shear systematics must be below
-one part in 10,000 for additive shear, and one part in 1000 for multiplicative shear.
-Each visit to a sky patch encounters these systematics. Some observing strategies can 
-effectively randomize these over all visits to a field.  Below we discuss the observing 
-strategies for suppressing shear systematics and metrics for their success.
+stringent requirements on our ability to accurately measure galaxy shapes and
+redshifts, which in turn demands precise and accurate knowledge of the point
+spread function, astrometry, and photometry.  These measurements are influenced
+by the interaction of light with the Earth's atmosphere, the telescope optics,
+and the CCD sensors.  Systematics in the shear are introduced in each case.
+Observing strategies have been developed for suppressing these systematics in
+current lensing surveys, such as the Deep Lens
+Survey\footnote{\url{dls.physics.ucdavis.edu}}.  These and new methods will be
+applied to the LSST survey.
 
-\subsection{Target Selection}
+To leading order, we can express the effect of shear systematics in the observed
+shear $\gamma^\mathrm{obs}_i$ as a small linear perturbation of the true shear
+$\gamma_i$,
 
-Image quality must be uniformly good in the bands used for weak lens shear.  These will be 
-the $r$ and $i$ bands.   Depending on the current weather and seeing, the scheduler
-will have a list of priorities for next-field, based on prior history of coverage.
-Nearby fields in need of coverage in these bands will be given high priority if the
-seeing is better than some specified value, likely 0.7 arcsec FWHM.
+$$ \gamma_i^\mathrm{obs} = (1+m_i) \gamma_i + c_i, $$
 
+where $m_i$ is the multiplicative and $c_i$ is the additive systematic in the
+ith shear component.  From the LSST SRD requirements on residual systematics in
+the shear-shear correlation function one can specify the level of residual shear
+systematics at which statistical uncertainties become subdominant.  Over the
+sample of 3-4 billion galaxies, the shear systematics must be below 3 parts in
+10,000 for additive shear $|c|$, and 3 parts in 1000 for multiplicative shear
+$|m|$.  Each visit to a sky patch encounters these systematics.  In particular,
+each re-visit to a given field generates the same CCD-based additive shear
```

Comment:
No.  Or perhaps I failed to explain.  The CCD systematics are block 
printed on the sky in each revisit.  Dithering in xy helps only a 
little. Dithering in angle helps a lot.

Atmosphere systematics are indeed larger than the CCD systematics per 
visit but they are random and average down over many revisits.


On 6/29/16 6:48 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69058231>:
>
>>
>> +where $m_i$ is the multiplicative and $c_i$ is the additive systematic in the
>> +ith shear component.  From the LSST SRD requirements on residual systematics in
>> +the shear-shear correlation function one can specify the level of residual shear
>> +systematics at which statistical uncertainties become subdominant.  Over the
>> +sample of 3-4 billion galaxies, the shear systematics must be below 3 parts in
>> +10,000 for additive shear $|c|$, and 3 parts in 1000 for multiplicative shear
>> +$|m|$.  Each visit to a sky patch encounters these systematics.  In particular,
>> +each re-visit to a given field generates the same CCD-based additive shear
>
> That's only true if we re-visit without dithering at all. If we dither,
> then the galaxies will experience a different part of any CCD-based
> additive shear.
>
> Also - aren't additive systematics of atmospheric and optical origin
> dominant over CCD-based additive shears?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69058231>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6YjDrvQdyUJ29lvf6KuXC41rI8tBDks5qQyBmgaJpZM4I9hd->.
>